### DISCUSSION TITLE
Support source code in function template

### DISCUSSION THREAD

**Discussion by kg-intec** [JakubAndrysek/MkDoxy#42] [Ideas]
Hello,

doxygen supports the showing of source code inplace of function definition: [Example](https://github.com/kg-intec/images/blob/main/code.png)

It would be nice to get a simmilar detailed view with call graph and source code.

**Comment by JakubAndrysek** [comment]
Thank you for your feedback, I will add it to my TODO list.

**Comment by JakubAndrysek** [comment]
Added
